src/test/regress/citus_tests/query_generator/generate_queries.py

```diff
@@ -54,6 +71,13 @@ def _fileMode(ddls, data):
 if __name__ == "__main__":
     signal.signal(signal.SIGINT, _signal_handler)
 
+    arguments = docopt(__doc__, version="generate_queries")
+    seed = random.randint(1, 1000000)
```

Comment:
why do we generate the value even if the user provided it? it seems a little confusing

Reply:
`seed` is optional argument which can be set by us to reproduce a previous scenario. If not set, we randomly generate a and use new seed.

Reply:
Instead of using a random number as the seed, it is imho more common to use the current time in milliseconds or sth as the seed. The range here is pretty minimal and limits the effective permutations we test due to the birthday problem, while the ms unix time is every exploring new seeds. just my 2c